src/dbos-executor.ts

```diff
@@ -258,7 +266,7 @@ export class DBOSExecutor {
     for (const ro of registeredClassOperations) {
       if (ro.workflowConfig) {
         const wf = ro.registeredFunction as Workflow<any, any>;
-        this.#registerWorkflow(wf, ro.workflowConfig);
+        this.#registerWorkflow(wf, {registration: ro, ...ro.workflowConfig});
```

Comment:
`registration` not super idiomatic. The meaning is "registration object"?

Reply:
That's not my biggest complaint.  The tracking structures feel a bit... "organic".